docs/use-cases/tokenizations.page.tsx

```diff
@@ -0,0 +1,310 @@
+    descriptions: [
+      {
+        subtitle: "Freeze:",
+        description:
+          "If you see signs of suspicious activity, you can suspend trading of your token while investigating the issue.",
+      },
+      {
+        subtitle: "Clawback:",
+        description:
+          "Recover tokens distributed to accounts in error: for example, reclaim funds sent to an account sanctioned for illegal activity.",
+      },
+    ],
+  },
+  {
+    title: "Nontransferable Tokens",
+    descriptions: [
+      {
+        subtitle: "",
+        description:
+          "Native support for nontransferable items such as identity tokens, airline credits, and consumer rewards, honored by all on-chain participants.",
+      },
+    ],
+  },
+  {
+    title: "Royalties",
+    descriptions: [
+      {
+        subtitle: "Secures Creator Compensation:",
+        description:
+          "Reliably collect your percentage of the sale price of your tokens.",
+      },
+    ],
+  },
+];
+
+export default function XrplOverview() {
+  const { translate } = useTranslate();
+  const modalRef = React.useRef(null);
+
+  return (
+    <div className="landing">
+      <div className="position-relative"></div>
+      <div className="position-relative d-none-sm"></div>
+      <section className="container-new py-26">
+        <div className="card-grid card-grid-2xN">
+          <div className="col">
+            <img
+              className="mw-100"
+              id="tokenizations-graphic"
+              alt="(Graphic: Tokenizations)"
+              src={require("../../static/img/backgrounds/tokenization-illustration.svg")}
+            />
+          </div>
+          <div className="col">
+            <div className="d-flex flex-column-reverse">
+              <h2 className="h4 h2-sm mb-8">
+                {translate(
+                  "Work with a variety of tokens supported by the XRP Ledger."
+                )}
+              </h2>
+              <h6 className="eyebrow mb-3">{translate("Tokenization")}</h6>
+            </div>
+            <p className="mb-4">
+              {translate(
+                "Explore the possibilities of tokenization on the XRP Ledger, revolutionizing ownership, transactions, and value exchange with unparalleled efficiency and security."
+              )}
+            </p>
+            <img
+              id="tokenizations-color-bar"
+              alt="Color Bar Graphic"
+              src={require("../../static/img/backgrounds/tokenization-color-bar.png")}
+            />
+            {useCases.map((useCase, index) => (
+              <div key={index} className="tokenization-use-case">
+                {useCase.name}
+                <button className="arrow-button" id="rightArrow">
+                  <img
+                    alt="right arrow"
+                    src={require("../../static/img/icons/arrow-right-purple.svg")}
+                  />
+                </button>
+              </div>
+            ))}
```

Comment:
I believe this element is supposed to be a nav list similar to ones we have on the XRP Overview and Docs landing pages. Of course, the links (as noted in my comment on the `useCases` array up above) are missing, and we should be sure to fill those in.

It would probably be best to create and use a shared React component for all of these. It's especially a little weird here since it's _almost_ but _not quite_ the same in terms of spacing and typesetting, and it would be good to make them consistent.

Side-by-side comparison:

| Previous Example | This Element |
|---|---|
| ![(cropped screenshot of a list from the Docs landing)](https://github.com/XRPLF/xrpl-dev-portal/assets/7515597/edd86344-598a-4740-87b1-b6bbc34c15a9) | ![(cropped screenshot of the use cases list from this page)](https://github.com/XRPLF/xrpl-dev-portal/assets/7515597/4c1067c7-4b0e-4813-9bdb-c1608f6f9eb3) |


Here's the source for those other examples:

- [XRP exchanges list](https://github.com/XRPLF/xrpl-dev-portal/blob/304ed0046ac13da6bc8a1406cb6cfc97eec575f0/about/xrp.page.tsx#L480-L496)
- [Docs landing use cases list](https://github.com/XRPLF/xrpl-dev-portal/blob/304ed0046ac13da6bc8a1406cb6cfc97eec575f0/docs/index.page.tsx#L167-L176) (also "Browse by Recommended Pages" in the same file)

Also, with the way this is coded right now, it re-uses `id="rightArrow"` for every element, which is against the rules of HTML (`id` attributes are supposed to be unique within a page).

Lastly, this is a nitpick, but I find it surprising that "Learn more" would take me to a YouTube video. Maybe it would provide a better user experience to add a little video camera icon as kind of a warning? We can quickly do this with `<i class="fa fa-video-camera"> </i>` which would look like the following:

![(screenshot of "Learn more" link prefaced with a video camera icon)](https://github.com/XRPLF/xrpl-dev-portal/assets/7515597/275726c1-2d68-4477-a9d8-8d4fe4d91760)